src/pages/Dashboard/inputComponent.tsx

```diff
@@ -0,0 +1,36 @@
+// import React from "react";
+// import { useForm, UseFormRegister, FieldValues } from "react-hook-form";
+import { ReusableInputProps } from "./inputComponent.definitions";
+import "./inputComponent.css";
+
+const ReusableInput = ({
+  name,
+  register,
+  errors,
+  isRequired,
+  label,
+  placeholder,
+  defaultValue,
+  errorText,
+}: ReusableInputProps) => {
+  return (
+    <div className="form-field">
+      <div className="label-and-error">
+        <p className="label">{label}</p>
+        <p className="error-text">{errors[name] && errors[name].message}</p>
```

Comment:
Lets have the error just be a string without the parenthesis. The we can put the in here like this:

```<p className="error-text">{"(" + errors[name]?.message + ")"}</p>```